recipes/jinjasql/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "jinjasql" %}
+{% set version = "0.1.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5ba8fc1ce0c037da0b3e122d4bbc7b8fd47e1fa73ec72ef72c4c495f81f042a1
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - jinja2 >=2.5
+    - django >=1.7,<1.8
```

Comment:
It's in the `requirements.txt`, but `django` doesn't appear to be required:

https://github.com/hashedin/jinjasql/issues/8

(old issue, but I did a search of the code). Same goes for `pyyaml`. Just appears to be used in the tests.

Reply:
fixed!